### PR TITLE
A-AddCar

### LINKED ISSUE #57: Implement support for adding cars
Adds a new Car object to an ArrayList storing Car objects.

Every car object has the following attributes:
- Model name
- License Plate Number
- Rental Price

### LINKED ISSUE #58: Implement 'CarParser' to parse user input into 'Car' object
The CarParser class:
1. Checks if format specified is valid.
2.  Parses the valid user input into a Car object and returns a Car object.

### LINKED ISSUE #59: Add simple exception for invalid 'add-car' command format.

### PR DESCRIPTION
**Pull request by CT9ARyan**
Add 'Car' objects to an ArrayList of 'Cars' Fixes #57 

Implement 'Parser' class to parse user input.

Implement 'CarParser' class to parse valid user input into a 'Car' object. Fixes #58 

Add simple exception for invalid 'add-car' command format. Fixes #59

### CODE REVIEW

File: src/main/java/parser/CarParser.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package parser;
+
+import car.Car;
+import exceptions.CarException;
+
+public class CarParser {
```
Comment (tkhahns) [inline src/main/java/parser/CarParser.java:6]: It's a good idea to separate the Parser for each class. Keep up with it!

File: src/main/java/parser/Parser.java [hunk 2]
```diff
@@ -1,91 +1,46 @@
...
-            if (!userInput.contains(parameter)) {
-                return false;
-            }
-        }
-
-        for(int i = 1 ; i < parameters.length ; i++){
-            if(userInput.indexOf(parameters[i]) < userInput.indexOf(parameters[i-1])){
-                return false;
-            }
+    public static boolean parse(String userInput) {
```
Comment (tkhahns) [inline src/main/java/parser/Parser.java:26]: Following this method, I believe everything will be more organized as we will have a main Parser to call each separate class' parser

File: src/test/java/parser/ParserTest.java [hunk 3]
```diff
@@ -8,18 +8,18 @@ public class ParserTest {
 
     @Test
     void parseParameterContents() {
-        Parser.userInput = "add-user /u test /a 18 /c 92750174";
+        String userInput = "add-user /u test /a 18 /c 92750174";
```
Comment (tkhahns) [inline src/test/java/parser/ParserTest.java:11]: Later you can add test cases for add-car also

**Review by tkhahns — COMMENTED** [review]
I believe everything is good overall and no need for major changes. Keep up the good work, Ryan!

**Review by rexkoh425 — APPROVED** [review]
LGTM

**Pull request merged**